src/test/java/org/broadinstitute/consent/http/authentication/OAuthAuthenticatorTest.java

```diff
@@ -31,16 +34,17 @@ class OAuthAuthenticatorTest {
   private SamService samService;
   private OAuthAuthenticator oAuthAuthenticator;
   private final ClaimsCache headerCache = ClaimsCache.getInstance();
+  private final String bearerToken = RandomStringUtils.randomAlphabetic(100);
+  private final MultivaluedMap<String, String> headerMap = new MultivaluedHashMap<>();
 
   @BeforeEach
   void setUp() {
     headerCache.cache.invalidateAll();
+    headerMap.clear();
```

Comment:
This is unnecessary as a new map is created for each test (method) run. If the map was `static` then this would be needed.

In the case of `headerCache`, `invalidateAll()` is needed because `ClaimsCache` has static state.